types/fake-response/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-redundant-jsdoc": true
+    }
+}
```

Comment:
```suggestion
{
    "extends": "dtslint/dt.json"
}
```
please apply my comments, amedn your tests, run local lint. Your package is ready to be merged and passes tests on updated branch:
```bash
❯ npm test

> definitely-typed@0.0.3 test C:\github\DefinitelyTyped
> node node_modules/@definitelytyped/dtslint-runner/dist/index.js --path .

Using local DefinitelyTyped at .
Parsing definitions...
Found 7029 packages.
Parsing in 12 child process(es)...
Parsing took 20.177 s
Running: git rev-parse --verify master
f03d6da5b9ab098580600ed2d870a47dfcb3fb05
Running: git diff master --name-status
A       types/fake-response/fake-response-tests.ts
A       types/fake-response/index.d.ts
A       types/fake-response/model.d.ts
A       types/fake-response/tsconfig.json
A       types/fake-response/tslint.json
Testing 1 changed packages: fake-response
Testing 0 dependent packages:
Installing NPM dependencies...
  ./types/node: npm install --ignore-scripts --no-shrinkwrap --no-package-lock --no-bin-links --no-save
 from ./types/node: up to date in 0.943s
found 0 vulnerabilities
1> fake-response START
1> fake-response OK
```